modules/ROOT/pages/data-integrity.adoc

```diff
@@ -0,0 +1,269 @@
+This action is crucial to limit the number of transactions that need to be replayed during the recovery process, particularly in order to minimize the time required for recovery after an improper shutdown.
+
+Despite the presence of checkpoints, database operations remain secure, as any transactions that have not been confirmed to have their modifications persisted to storage will be replayed upon the next database startup.
+However, this assurance is contingent upon the availability of the collection of changes comprising these transactions, which is maintained in the transaction logs.
+
+Maintaining a long list of unapplied transactions (due to infrequent checkpoints) leads to the accumulation of transaction logs, as they are essential for recovery purposes.
+Checkpointing involves the inclusion of a special "Checkpointing" entry in the transaction log, marking the last transaction at which checkpointing occurred.
+This entry serves the purpose of identifying transaction logs that are no longer necessary, as all the transactions they contain have been securely stored in the storage files.
+
+The process of eliminating transaction logs that are no longer required for recovery is known as pruning. From the aforementioned explanation, it becomes evident that pruning is reliant on checkpointing.
+In other words, checkpointing determines which logs can be pruned and determines the occurrence of pruning, as the absence of a checkpoint implies that the set of transaction log files available for pruning cannot have changed.
+Consequently, pruning is triggered whenever checkpointing takes place, with or without a specific verification of their existence.
+
+=== Triggering of checkpointing (and pruning) events
+
+The checkpointing policy, which is the driving event for pruning is configured by xref:configuration/configuration-settings.adoc#config_db.checkpoint[`db.checkpoint`] and can be triggered in a few different ways:
+
+* `PERIODIC` label:default[Default]
++
+By default, this policy checks every 15 minutes whether there are changes pending flushing (i.e. transactions that have not been checkpointed yet).
+If so, it performs a checkpoint and subsequently triggers a log prune.
+Note that no checkpointing is being performed implying no pruning happens.
+This is the default behavior and the only one available in Community Edition.
+
+* `CONTINUOUS` label:enterprise[Enterprise Edition]
+This policy constantly checks if a checkpoint is possible (i.e if any transactions committed since the last successful checkpoint) and if so, it performs it.
+* Pruning is triggered immediately after it completes, just like in the periodic policy.
+
+* `VOLUMETRIC` label:enterprise[Enterprise Edition]
+This checkpointing policy checks every 10 seconds if any logs are available for pruning and, if so, it triggers a checkpoint and subsequently, it prunes the logs.
```

Comment:
This policy has been reworded to:
This policy checks every 10 seconds if there is enough volume of logs available for pruning and, if so, it triggers a checkpoint and subsequently, it prunes the logs.
By default, the volume is set to 250MiB, but it can be configured using the settings xref:configuration/configuration-settings.adoc##config_db.tx_log.rotation.retention_policy[`db.tx_log.rotation.retention_policy`] and xref:configuration/configuration-settings.adoc#config_db.tx_log.rotation.size[`db.tx_log.rotation.size`].
See <<control-log-pruning, Control transaction log pruning>> for more details.